AniListAPI/Sources/LocalCacheMutations/BasicMediaListEntry.graphql.swift

```diff
@@ -113,7 +113,7 @@ public struct BasicMediaListEntry: AniListAPI.MutableSelectionSet, Fragment {
         "completedAt": completedAt._fieldData,
       ],
       fulfilledFragments: [
-        ObjectIdentifier(BasicMediaListEntry.self)
+        ObjectIdentifier(Self.self)
```

Comment:
what's your apollo version? I think you should update the CLI by installing it again. This [changes](https://github.com/axiel7/AniHyou/commit/f2e0c86b57a24438bfdb3d83c51890a9b9bbd8f5) were caused by updating Apollo to 1.2.2

Reply:
xcode 😬  yeah I think it decided 1.2.1 was the one it should use. i'll fix it